### PR TITLE
docs: publish v0.4 feature docs

### PR DESCRIPTION
**Pull request by stijnmoreels**
Publishes the v0.4 feature docs.

### CODE REVIEW

**Comment by stijnmoreels** [comment]
NuGet package not yet released, but already preparing v0.4 feature docs so we can verify PR #141 .

File: docs/versioned_docs/version-v0.4.0/01-index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# Installation
+
+The Arcus BackgroundJobs can be installed via NuGet:
+
+```shell
+PM > Install-Package Arcus.BackgroundJobs.CloudEvents
+```
+
+For more granular packages we recommend reading the documentation.
```
Comment (fgheysels) [inline docs/versioned_docs/version-v0.4.0/01-index.md:19]:
```suggestion
For more in depth documentation we recommend reading the documentation.
```
I'd also put a link to the documentation ?

**Review by fgheysels — APPROVED** [review]

File: docs/versioned_docs/version-v0.4.0/01-index.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v0.4.0/01-index.md:19]: Hmm, but here we're already at the documentation site 😅 . Maybe we could add `:` at the end as to link to the `# Features` title below.

Comment (stijnmoreels) [reply docs/versioned_docs/version-v0.4.0/01-index.md]: Removed this sentence and NuGet badge in https://github.com/arcus-azure/arcus.backgroundjobs/pull/143/commits/a611c6a3123ab76aeaeaa817b9edce9df8fb7758

**Pull request merged**